src/services/scan_form_service.js

```diff
@@ -8,17 +8,35 @@ export default (easypostClient) =>
 
     /**
      * Create a ScanForm.
-     * @param {object} prams
+     * @param {object} params
      * @returns {ScanForm}
      */
     static async create(params) {
-      try {
-        // TODO: We should re-implement the check here that wraps up params in `shipments` if the user didn't
-        const response = await easypostClient.post(this._url, params);
+      // TODO: We should re-implement the check here that wraps up params in `shipments` if the user didn't
+      const url = `${this._url}`;
 
-        return this.convertToEasyPostObject(response.body);
-      } catch (e) {
-        return Promise.reject(e);
-      }
+      const wrappedParams = {};
+      wrappedParams[this.key] = params;
+
+      return this._create(url, wrappedParams);
+    }
+
+    /**
+     * Retrieve a list of all scan forms associated with the API key.
+     * @param {object} params
+     * @returns {ScanForm[]}
```

Comment:
Reminder that list of objects isn't correct throughout this PR since the results are paginated